modules/op-parameters-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -1,11 +1,16 @@
 // This module is included in the following assemblies:
-// * pac/managing-pipeline-runs-pac.adoc
+// * pac/creating-pipeline-runs-pac.adoc
 
 :_mod-docs-content-type: REFERENCE
 [id="parameters-pipeline-run-using-pipelines-as-code_{context}"]
-= Parameters in a pipeline run specification
+= Dynamic variables in a pipeline run specification
 
-You can use parameters in a pipeline run specification to provide information about the commit that triggered the pipeline run and to use the temporary GitHub App token for Github API operations.
+You can use dynamic variables in a pipeline run specification to provide information about the commit that triggered the pipeline run and to use the temporary GitHub App token for Github API operations.
```

Comment:
this looks very short compared how detailed the upstream documentation is, this missed information about

* Access nested fields within {{ body }} (e.g., {{ body.pull_request.title }}).
* Access specific headers using bracket notation (e.g., {{ headers['x-github-event'] }}).
*  Handle complex or multi-line dynamic variable values ({{ body }}, {{ trigger_comment }}) within YAML parameters using block format (|- or >) to avoid parsing errors, as mentioned in the upstream documentation.
* Provide practical examples of processing {{ body }} or {{ headers }} content within task scripts (like the upstream example using Python for labels).

Reply:
Discussed this ijn chat, putting here just to have a complete record. The scope of this procedure is intentionally very basic. We do need to improve reference information too and I have added these notes to https://issues.redhat.com/browse/RHDEVDOCS-6422 where I track further improvement needs.